content/en/serverless/azure_app_service/windows_code.md

```diff
@@ -551,16 +551,15 @@ It is likely that you do not have the Azure integration configured to monitor yo
 
 **Note**: To expedite the process of investigating application errors with the support team, set `DD_TRACE_DEBUG:true` and add the content of the Datadog logs directory (`%AzureAppServiceHomeDirectory%\LogFiles\datadog`) to your email.
 
-Still need help? Contact [Datadog support][6].
+Still need help? Contact [Datadog support][5].
 
 ### Further Reading
 
 {{< partial name="whats-next/whats-next.html" >}}
 
 [1]: /developers/dogstatsd
-[2]: /tracing/setup/dotnet/
```

Comment:
This `[2]: /tracing/setup/dotnet/` link wasn't used, so I removed it and renumbers the rest.